plugins/woocommerce-blocks/tests/e2e/utils/shipping/shipping-utils.page.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { Page } from '@playwright/test';
+import { Admin } from '@wordpress/e2e-test-utils-playwright';
+
+export class ShippingUtils {
+	private page: Page;
+
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	async openShippingSettings( { admin }: { admin: Admin } ) {
+		await admin.visitAdminPage(
+			'admin.php',
+			'page=wc-settings&tab=shipping&section=options'
+		);
+	}
+
+	async enableShippingCalculator( { page }: { page: Page } ) {
```

Comment:
I think that expecting `page` to be passed in args is wrong, `this.page` should be used instead and the arg can be removed. The util should also open the admin page and save the setting.

Ideally, the dev would only have to call: `shippingUtils.enableShippingCalculator()` and that would take care of everything. Same for all other utils using `page` in the args.

Reply:
Thanks for this great suggestion. I've adjusted the PR accordingly, and also adjusted all other similar sections.